definitions/firefox_desktop.toml

```diff
@@ -21,6 +21,15 @@ description = """
 select_expression = '{{agg_sum("active_hours_sum")}}'
 data_source = "clients_daily"
 
+[metrics.active_hours_glean]
+friendly_name = "Active hours in Glean"
+description = """
+    Measures the amount of time (in 5-second increments) during which
+    Firefox received user input from a keyboard or mouse. The Firefox
+    window does not need to be focused.
+"""
+select_expression = '{{agg_sum("browser_engagement_active_ticks/(3600/5)")}}'
```

Comment:
What does the `/5` at the end do? I can understand dividing by 3600 (to convert to hours) but I'm not clear on what the additional division is for.

Reply:
Yes, it is. The active_hours_sum in legacy that was used for the active_hours metrics is defined same as this one: https://github.com/mozilla/bigquery-etl/blob/704b6d579c1ba82487c8d7d008d74ebb27a3b65f/sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/query.sql#L751